plugins/user_quota/server/quota.py

```diff
@@ -174,63 +174,55 @@ def _validatePolicy(self, policy):
                            dictionary as if the JSON was already decoded.
         :returns: a validate policy dictionary.
         """
-        validKeys = []
-        for key in dir(self):
-            if key.startswith('_validate_'):
-                validKeys.append(key.split('_validate_', 1)[1])
-        for key in list(policy):
-            if key.startswith('_'):
-                del policy[key]
+        validKeys = [k[10:] for k in dir(self) if k.startswith('_validate_')]
+        policy = {k: v for k, v in six.iteritems(policy) if not k.startswith('_')}
```

Comment:
@manthey Can you provide any insight as to [why this check was added](https://github.com/girder/girder/commit/0bb4cee20be050c99b17450fff51c719ef388af8#diff-9e7feac579f760b14c6480f3d8477a83R184)? It'd be good to have a code comment here explaining it.

Reply:
If I recall, we add _ prefixed values to resource dictionaries (with, for instance, `_currentFileSizeQuota`), but we don't want to write that back to the database, so this strips out those private values.